arangod/Cache/TransactionManager.h

```diff
@@ -70,14 +70,19 @@ class TransactionManager {
   std::uint64_t term();
 
  private:
-  std::atomic<std::uint64_t> _openReads;
-  std::atomic<std::uint64_t> _openSensitive;
-  std::atomic<std::uint64_t> _openWrites;
-  std::atomic<std::uint64_t> _term;
-  std::atomic<bool> _lock;
+  struct Counters {
+    uint64_t openReads : 21;
+    uint64_t openWrites : 21;
+    uint64_t openSensitive : 21;
+  };
+  static_assert(sizeof(Counters) == sizeof(uint64_t), "unexpected size");
 
-  void lock();
-  void unlock();
+  struct alignas(16) Data {
+    Counters counters;
+    uint64_t term;
+  };
+
+  std::atomic<Data> _data;
```

Comment:
I know this is nit-picky, but could you rename `Data` to `State` and `_data` to `_state`?

Reply:
Sure! I am also an advocate for good names, so I actually appreciate comments like this - and `State` is indeed a better name here..! :-)